text/3-entrees/currency-imbalances.md

```diff
@@ -0,0 +1,36 @@
+
+# Currency Imbalances
+
+`pallets/currency-imbalances`
```

Comment:
You need to include the two badges here. Look at some of the other recipes writeups for an example.

Reply:
Please check it because it is done.